chips/nrf51/Cargo.toml

```diff
@@ -8,6 +8,7 @@ build = "build.rs"
 [dependencies]
 cortexm0 = { path = "../../arch/cortex-m0" }
 kernel = { path = "../../kernel" }
+capsules = { path = "../../capsules" }
```

Comment:
Why do you need a connection from the `chips` crate to the `capsules` crate? This seems wrong.